src/haz3lweb/Update.re

```diff
@@ -172,6 +172,9 @@ let reevaluate_post_update =
   | Undo
   | Redo => true;
 
+/* This functions returns a new model with updated results.
+   Editors.get_spliced_elabs generates the DHExp of the editor
```

Comment:
The comment on a function shouldn't reveal its internal implementation details -- you can leave those comments directly in the code below if you think it isn't self-explanatory from reading the comments on those functions